docker/create_new_docker_image.py

```diff
@@ -92,33 +137,17 @@ def main():
     base_image_last = "{}:{}".format(base_image, last_tag)
     print("Latest base image: " + base_image_last)
     os.mkdir(folder)
-    docker_template = DOCKER_ALPINE if args.linux == 'alpine' else DOCKER_DEBIAN
-    docker_file = open(folder + "/Dockerfile", "w+")
-    docker_file.write(docker_template.format(image=base_image_last))
-    docker_file.close()
     conf_file = open(folder + "/build.conf", "w+")
     conf_file.write("version=1.0.0\n")
     conf_file.close()
-    # copy gitignore from python image
-    shutil.copy(sys.path[0] + "/python/.gitignore", folder)
-    print("Initializing pipenv...")
-    print('========================================')
-    pipenv_param = "--two" if args.python == "two" else "--three"
-    my_env = os.environ.copy()
-    my_env['PIPENV_MAX_DEPTH'] = '1'
-    subprocess.call(["pipenv", pipenv_param], cwd=folder, env=my_env)
-    if args.pkg:
-        print("Installing python packages: {}".format(args.pkg))
-        cmd_arr = ["pipenv", "install"]
-        cmd_arr.extend(args.pkg)
-        subprocess.call(cmd_arr, cwd=folder, env=my_env)
+    if args.type == "python":
+        create_python_image(folder, base_image_last, args)
     else:
-        subprocess.call(["pipenv", "lock"], cwd=folder, env=my_env)
+        create_powershell_image(folder, base_image_last, args)
     print('Adding: {} to .dependabot/config.yml ...'.format(folder))
     subprocess.check_call([sys.path[0] + "/add_dependabot.sh", "docker/" + args.name])
```

Comment:
we can pass here the type as well so it wont print some thing `Not adding python dependency config as docker/<IMAGE NAME>/Pipfile not found` for powershell images

Reply:
Let's leave this to a separate PR.